articles/20240822_v0.24.0_news.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Daytona v0.24.0"
+description: "Explore the new features, bug fixes, and improvements in Daytona v0.25.0"
+date: 2024-08-22
+author: "Dev Gajjar"
+tags: ["Release Notes", "v0.24.0"]
+---
+
+# Daytona v0.24.0 Release Summary
+
+**Watch the Full Office Hours Recap on YouTube:** [Daytona Office Hours #7](https://www.youtube.com/watch?v=nVQWa4jmwLc&t=622s)
+
+## Overview
+
+The Daytona Office Hours #7 introduced several updates in the v0.25. This release focuses on enhancing user experience, improving project management, and extending compatibility with additional Git providers.
```

Comment:
DOH didn't introduce updates, that was done before, but DOH serves to showcase and explain to the community what was done over the last week, sometimes we even don't have new version in the meantime